evalml/pipelines/utils.py

```diff
@@ -369,6 +387,119 @@ def _make_new_component_name(model_type, component_name, idx=None):
     )
 
 
+def _make_pipeline_from_multiple_graphs(
+    input_pipelines,
+    estimator,
+    problem_type,
+    parameters={},
```

Comment:
I think mutable defaults are dangerous, let's use None for the default?

Reply:
Good call!